python/src/scipp/__init__.py

```diff
@@ -25,7 +26,7 @@ def custom_formatwarning(msg, *args, **kwargs):
 from ._scipp import __version__
 # Import classes
 from ._scipp.core import Variable, DataArray, Dataset, GroupByDataArray, \
-                         GroupByDataset, Unit
+                         GroupByDataset, Unit, Coords, Masks
```

Comment:
Is this useful (or used)? If not revert?

Also wodnering why the `GroupBy*` classes are imported? If only for setting extra properties below...

Reply:
Yes, I was setting a method on the `Coords` and `Masks` before, but not anymore.
`GroupBy*` are imported only for setting the extra properties, yes.